dotnet-server-sdk-shared-tests/LaunchDarkly.ServerSdk.SharedTests.Tests/BigSegmentStore/BigSegmentStoreBaseTestsTest.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using LaunchDarkly.Sdk.Server.Interfaces;
+using Xunit.Abstractions;
+
+using static LaunchDarkly.Sdk.Server.Interfaces.BigSegmentStoreTypes;
+
+namespace LaunchDarkly.Sdk.Server.SharedTests.BigSegmentStore
```

Comment:
This is the shared test code from https://github.com/launchdarkly/dotnet-server-sdk-shared-tests/pull/3; you can skip everything in this directory since it's already being reviewed in that PR.